README.md

```diff
@@ -1,206 +1,68 @@
+| [verification-by-reference.md](./requirements/verification-by-reference.md) | Requirement of verification by reference |
 
-![Notary v2 scenarios](/media/notary-e2e-scenarios.svg)
+## Security Documents
 
-Notary v2 provides for multiple signatures of an [OCI Artifact][oci-artifacts] (including container images) to be persisted in an [OCI conformant][oci-distribution-conformance] registry.
-Artifacts are signed (`notation sign`) with private keys, and validated with public keys (`notation verify`).
-To support user deployment flows, signing an OCI Artifact will not change the `@digest` or `artifact:tag` reference.
-To support content movement across multiple certification boundaries, artifacts and their signatures will be easily copied within and across [OCI conformant][oci-distribution-conformance] registries.
+| File Name     | Description |
+| -------- | ----------- |
+| [ADA-notation-security-audit-23.pdf](./security/reports/audit/ADA-notation-security-audit-23.pdf) | Security audit report in 2023 covering [notation](https://github.com/notaryproject/notation), [notation-go](https://github.com/notaryproject/notation-go), and [notation-core-go](https://github.com/notaryproject/notation-core-go) repositories |
+| [ADA-fuzzing-audit-22-23.pdf](./security/reports/fuzzing/ADA-fuzzing-audit-22-23.pdf) | Fuzz testing audit in 2023 covering [notary](https://github.com/notaryproject/notaty), [notation-go](https://github.com/notaryproject/notation-go), and [notation-core-go](https://github.com/notaryproject/notation-core-go) repositories |
 
-![Notary v2 dependent projects](/media/oss-project-sequence.svg)
+## Specifications
 
-To deliver on the Notary v2 goals of cross registry movement of artifacts with their signatures, changes to several projects are anticipated, including [OCI distribution-spec][oci-distribution-spec], [CNCF Distribution][cncf-distribution], [OCI Artifacts][oci-artifacts], [ORAS][oras] with further consumption from projects including [containerd][containerd], [OPA][opa], [Gatekeeper][gatekeeper] and the [docker client][docker-client].
+| File Name     | Description |
+| -------- | ----------- |
+| [plugin-extensibility.md](./specs/plugin-extensibility.md) | Notation Plugin specification |
+| [signature-envelope-cose.md](./specs/signature-envelope-cose.md) | Notary Project OCI COSE signature envelope |
+| [signature-envelope-jws.md](./specs/signature-envelope-jws.md) | Notary Project OCI JWS signature envelope |
+| [signature-specification.md](./specs/signature-specification.md) | Notary Project OCI signature specification |
+| [signing-and-verification-workflow.md](./specs/signing-and-verification-workflow.md) | Notary Project OCI signing and verification workflow |
+| [signing-scheme.md](./specs/signing-scheme.md) | Notary Project signing scheme|
+| [trust-store-trust-policy.md](./specs/trust-store-trust-policy.md) | Notation Trust Store and Trust Policy  |
```

Comment:
@yizha1 @toddysm, the Trust Store and Policy spec should be part of Notary Project specs, it has the core verification logic that is referenced from the OCI specific workflow in [Signing and Verification Workflow](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-and-verification-workflow.md).

Reply:
@gokarnm Trust Store and Policy spec are still part of Notary Project specs at this moment. If I understood your comments correctly, the description `Notation Trust Store and Trust Policy` should be changed to `Notary Project Trust Store and Trust Policy`?

Reply:
@gokarnm @toddysm updated description to `Notary Project Trust Store and Trust Policy`. We can continuously discuss the content and folder structure here https://github.com/notaryproject/.github/issues/48

Reply:
@yizha1 yes, that was the change I was proposing. Thanks for the update!